default/stringtables/en.txt

```diff
@@ -15556,6 +15556,12 @@ ST_CLOAK_4_DESC
 
 Improves [[metertype METER_STEALTH]] of ship on which it is mounted by modulating the frequency of any matter waves emitted from the ship to match that of the background radiation. [[NO_STACK_STEALTH_SHIP_PARTS]]'''
 
+AR_REINFORCED_HULL
+Self-Reinforcing Hull
```

Comment:
Needs a better name. Maybe `Auto-Repair Bay` or `Hull Regenerator`

Reply:
can you propose one acceptable name?

i can perform refactorings when ever necessary and as often as desired. or i could make two module variations for both those proposed names, one stays core slot the other as internal slot?

Reply:
> can you propose one acceptable name?

I did in the comment you replied to...

Reply:
H.R. it is then

Reply:
Should be "Hull Regenerator" rather than "Self-Reinforcing Hull", as it's not actually a hull.